README.md

```diff
@@ -35,15 +36,19 @@ Generating a format plugin is as easy as running the following:
 
 Repeated for the "newline" challenged:
 
-    mvn archetype:generate -DarchetypeArtifactId=nexus-format-archetype -DarchetypeGroupId=org.sonatype.nexus.archetypes -DarchetypeVersion=1.0-SNAPSHOT -DgroupId=org.sonatype.nexus.repository -DartifactId=nexus-repository-foo -DpluginFormat=foo -DpluginClass=Foo -Dversion=0.0.1
+    mvn archetype:generate -DarchetypeArtifactId=nexus-format-archetype -DarchetypeGroupId=org.sonatype.nexus.archetypes -DarchetypeVersion=1.0-SNAPSHOT -DnexusPluginsVersion=3.18.0-01 -DgroupId=org.sonatype.nexus.repository -DartifactId=nexus-repository-foo -DpluginFormat=foo -DpluginClass=Foo -Dversion=0.0.1
```

Comment:
It looks like `nexusPluginsVersion` is also optional, maybe add this to the optional parameters description below with default

Reply:
> It looks like `nexusPluginsVersion` is also optional, maybe add this to the optional parameters description below with default

I added more docs about the required and optional parameters. 
To minimize the number of places we reference hard coded versions I also doc'd a linked to the source that defines the `defaultNexusPluginsVersion`. Hopefully that helps. 
efd8234